### PR TITLE
Translations updates from VIVO-DE community

### PR DESCRIPTION
**Pull request by KitioFofack**

### CODE REVIEW

**Comment by awoods** [comment]
@KitioFofack : can you please add a link to the associated JIRA... or please create a JIRA if one does not exist?

**Comment by awoods** [comment]
JIRA: https://jira.duraspace.org/browse/VIVO-1661

File: webapp/src/main/webapp/i18n/all_de_DE.properties [hunk 1]
```diff
@@ -0,0 +1,910 @@
...
+enter_new_password = Bitte geben Sie Ihr neues Passwort für {0} ein.
+error_no_email_address = Bitte geben Sie Ihre E-Mail-Adresse ein.
+error_no_password = Bitte geben Sie Ihr Passwort ein.
+error_incorrect_credentials = Die E-Mail oder das Passwort ist falsch.
+logins_disabled_for_maintenance = Benutzer-Logins sind vorübergehend deaktiviert, während das System gewartet wird.
+error_no_new_password = Bitte geben Sie ihr neues Passwort ein.
+error_passwords_dont_match = Die eingegebenen Passwörter stimmen nicht überein.
+error_password_length = Bitte geben Sie ein Passwort zwischen {0} und {1} Zeichen Länge ein.
+error_previous_password = Ihr neues Passwort darf nicht dem Aktuellen entsprechen.
+
```
Comment (gneissone) [inline webapp/src/main/webapp/i18n/all_de_DE.properties:122]:
```suggestion
password_system_has_changed = Our password system has been upgraded to make it more secure. \
Please enter a new password.

```
Please add this string, it is now part of Vitro all.properties. See https://github.com/vivo-project/Vitro/commit/fea33a456a8755efebaaf4737526a1487ade3608#diff-4d548d21dbf1c2f5a307a6f2a1ce1882R126. Would be best if it was added in German :)

**Review by gneissone — CHANGES_REQUESTED** [review]
Installed fine, switched to German and appears to be working. There is still random English appearing in places but I figure that's a VIVO/Vitro issue, not one with this PR.

Please add the missing language string then I will approve.

File: webapp/src/main/webapp/i18n/all_de_DE.properties (hunk 1 above)
Comment (grahamtriggs) [reply webapp/src/main/webapp/i18n/all_de_DE.properties:122]: Added a translation for password migration.

**Review by gneissone — APPROVED** [review]

**Pull request merged**